lib/tools.js

```diff
@@ -86,7 +86,10 @@ function asJson (obj, msg, num, time) {
   if (hasObj === true) {
     var notHasOwnProperty = obj.hasOwnProperty === undefined
     if (objError === true) {
-      data += ',"type":"Error","stack":' + stringify(obj.stack)
+      data += ',"type":"Error"'
```

Comment:
do tests pass with the next-major branch of pino-pretty

Reply:
pino-pretty is passing: https://travis-ci.org/pinojs/pino-integration/jobs/400498917